src/main/java/bio/terra/service/load/flight/LoadLockStep.java

```diff
@@ -9,21 +9,24 @@
 import bio.terra.stairway.StepStatus;
 import java.util.UUID;
 
-// This step is meant to be shared by dataset and filesystem flights for locking the load tag.
-// It expects to find LoadMapKeys.LOAD_TAG in the parameters or working map.
-
 public class LoadLockStep implements Step {
   private final LoadService loadService;
+  private final UUID datasetId;
 
-  public LoadLockStep(LoadService loadService) {
+  /**
+   * This step is meant to be shared by dataset and filesystem flights for locking the load tag. It
+   * expects to find LoadMapKeys.LOAD_TAG in the parameters or working map.
+   */
```

Comment:
Top-level comments are more useful as constructor Javadoc -- we construct new Steps when we define our flights, so you're more likely to see this context in the flight that way in your IDE.